components/org.wso2.carbon.identity.api.server.input.validation/org.wso2.carbon.identity.api.server.input.validation.v1/src/main/resources/inputValidation.yaml

```diff
@@ -81,6 +81,51 @@ paths:
           $ref: '#/components/responses/Forbidden'
         '500':
           $ref: '#/components/responses/ServerError'
+  /validation-rules/revert:
+    delete:
+      tags:
+        - Revert Validation Rules
+      description: Revert validation rules of all fields to the default configuration.
+      operationId: revertValidationRules
+      responses:
+        '204':
+          description: Revert successful
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    patch:
```

Comment:
Since /revert is treated as a functional operation, it's better to support full or partial revert both with 
POST /validation-rules/revert